ivy/functional/frontends/paddle/tensor/linalg.py

```diff
@@ -144,3 +144,10 @@ def bmm(x, y, transpose_x=False, transpose_y=False, name=None):
 @to_ivy_arrays_and_back
 def matrix_power(x, n, name=None):
     return ivy.matrix_power(x, n)
+
+
+# cond
+@with_supported_dtypes({"2.4.2 and below": ("float32", "float64")}, "paddle")
```

Comment:
The paddle version is `2.5.0`
```suggestion
@with_supported_dtypes({"2.5.0 and below": ("float32", "float64")}, "paddle")
```

Reply:
> The paddle version is `2.5.0`

how are we coming up with the version?